bedrock/firefox/templates/firefox/welcome/page10.html

```diff
@@ -12,52 +10,82 @@
 
 {% block body_class %}welcome-page10{% endblock %}
 
+{% block experiments %}
+  {% if switch('welcome-10-experiment', ['en-US', 'en-CA']) %}
```

Comment:
Should `en-GB` be added here? (I noticed in the experiment view there are three English locales listed).

Reply:
I think it's only targeting North America so it's the view that needs fixing. I'll verify to be sure.